README.md

```diff
@@ -177,9 +182,239 @@ The service will skip the next startup until the previous backup run is complete
 
 ### Can multiple backup routines be performed simultaneously?
 
-The service uses the [asbackup](https://github.com/aerospike/aerospike-tools-backup) shared library, which is not currently thread safe. 
-Given this limitation, backup routines are performed in sequence. 
+The service uses the [asbackup](https://github.com/aerospike/aerospike-tools-backup) shared library, which is not currently thread safe.
+Given this limitation, backup routines are performed in sequence.
 
 ### Which storage providers are supported?
 
 The backup service supports AWS S3 or compatible (such as MinIO) and local storage.
+
+## Example requests and responses
+
+### Read configurations
+
+This section details how to fetch configurations for clusters, policies, and storage options. This is useful for setting up or verifying the configuration of your system.
+
+#### Get cluster configuration
+
+This endpoint returns the configurations of existing clusters, including the default cluster setup with seed nodes and credentials.
+
+Request:
+```http
+  GET {{baseUrl}}/v1/config/clusters
+```
+
+<details>
+    <summary>Response:</summary>
+
+```json
+{
+  "absDefaultCluster": {
+    "seed-nodes": [
+      {
+        "host-name": "host.docker.internal",
+        "port": 3000
+      }
+    ],
+    "credentials": {
+      "user": "tester",
+      "password": "psw"
+    }
+  }
+}
+```
+</details>
+
+#### Get routine configuration
+
+Retrieves the configured backup routines, including policies, source clusters, storage options, and scheduling.
+
+Request:
+```http
+GET {{baseUrl}}/v1/config/routines
+```
+
+<details>
+    <summary>Response:</summary>
+
+```json
+{
+  "routine1": {
+    "backup-policy": "keepFilesPolicy",
+    "source-cluster": "absDefaultCluster",
+    "storage": "local",
+    "interval-cron": "@yearly",
+    "namespaces": ["source-ns7"]
+  },
+  "routine2": {
+    "backup-policy": "removeFilesPolicy",
+    "source-cluster": "absDefaultCluster",
+    "storage": "local",
+    "interval-cron": "@yearly",
+    "namespaces": ["source-ns8"],
+    "set-list": ["backupSet"],
+    "bin-list": ["backupBin"]
+  }
+}
+```
+</details>
+
+#### Get storage configuration
+
+Shows the storage configurations available, including local and MinIO (S3) storage options.
```

Comment:
The description should be more specific. It returns all the configured storage endpoints.